### ISSUE TITLE
Installed but cant get terminal

### ISSUE THREAD

**Issue by Dal78** [michaelshmitty/esperpass#6]
[Hi,]

Ive written the firmware and can get the initial greeting screen but cant get cmd prompt and reset just gets me back to the same static splash screen.

https://imgur.com/ff7lZAk

**Comment by michaelshmitty** [comment]
Greetings! Thanks for trying it out, we need more testers to come back and share their experience.

In PuTTY, in the settings panel on the left under the category "Terminal" try fiddling with the settings "Implicit CR in every LF" and "Implicit LF in every CR". You can also try switching them both on.
Also try changing the "Local echo" setting.
Hit enter a few times after you've reset the ESP chip to see the result.
It's possible your commands don't include a line feed / carriage return automatically and therefore the cmd prompt interpreter doesn't know you're issuing a command.

Keep an eye out for the configuration settings in PuTTY. I find it sometimes takes some care to make sure your connections settings are saved / kept properly.

Let me know if you get it to work and if so which settings worked for you.
If possible try another terminal program.
Perhaps not practical for you but on MacOS or Linux you can also us the "screen" program like this: `screen /dev/ttyUSB0 115200` (replace /dev/ttyUSB0 with the correct device for your ESP on that system).
Good luck!

**Comment by Dal78** [comment]
So i got it to work by hitting return before i had hit reset.   After hitting reset the terminal is not responsive until i have reloaded the putty session.  Seems a bit fussy about giving a cmd every time.

Never mind its up now.  Just waiting for a hit.  I can see it joins the wifi and my 3ds is joining and leaving so presumably the relay its hitting has no-one on it it tag.

Just got to wait i guess to see if its working :)

**Comment by michaelshmitty** [comment]
> might be good if we can have a command to cycle mac's immediately for testing.

There was a command in the original firmware called `ap_mac` but I deleted it in a somewhat overzealous ripping out of features frenzy. I figured it wasn't necessary because of the cycling, but you're right it would be convenient to put it back for testing purposes. Let me see if I can quickly fix that because it might help getting some clarity on #5 as well.
Considering this issue resolved! Thanks for following up on it. Happy tagging!
